src/main/java/htsjdk/variant/variantcontext/Allele.java

```diff
@@ -200,6 +200,11 @@ protected Allele(final Allele allele, final boolean ignoreRefState) {
     public final static Allele NO_CALL = new Allele(NO_CALL_STRING, false);
     public final static Allele NON_REF_ALLELE = new Allele(NON_REF_STRING, false);
 
+    public final static Allele SV_SIMPLE_DEL = new Allele(createBracketedSymbAlleleString(StructuralVariantType.DEL), false);
+    public final static Allele SV_SIMPLE_INS = new Allele(createBracketedSymbAlleleString(StructuralVariantType.INS), false);
+    public final static Allele SV_SIMPLE_INV = new Allele(createBracketedSymbAlleleString(StructuralVariantType.INV), false);
+    public final static Allele SV_SIMPLE_CNV = new Allele(createBracketedSymbAlleleString(StructuralVariantType.CNV), false);
+    public final static Allele SV_SIMPLE_DUP = new Allele(createBracketedSymbAlleleString(StructuralVariantType.DUP), false);
```

Comment:
Can you add a comment above each SV allele declaration, describing what kind of variant it represents?